lib/host_kafka.py

```diff
@@ -0,0 +1,33 @@
+import socket
+from contextlib import closing
+
+from app.config import Config
+from app.environment import RuntimeEnvironment
+from app.logging import get_logger
+
+logger = get_logger(__name__)
+
+
+def _any_bootstrap_server_connects(kafka_socket, servers) -> bool:
+    if not servers:
+        config = Config(RuntimeEnvironment.SERVICE)
+        servers = [config.bootstrap_servers]
```

Comment:
they're already a list
```suggestion
        servers = config.bootstrap_servers
```

Reply:
When there is a single Kafka server, a string is returned as `host:port`.  `[]` has been added to deal with such cases.

Reply:
That will make a double nested given a list though right? Should it conditionally cast to a list like
```
servers = [config.bootstrap_servers] if config.bootstrap_servers is string else config.bootstrap_servers
```
or something like that?

Reply:
Updated the code add suggested for being elegant.  Reason behind the previous implementation was `[[...]]` evaluated as string.  Here is how it was checked:
```
>>> fruit = [["apples", "oranges", "apricots"]]
>>> type(fruit)
<class 'list'>
>>> 
```